shared/metadata.go

```diff
@@ -28,9 +28,14 @@ type MetadataResult struct {
 	// Test is the name of a test; this often corresponds to a test file path in
 	// the WPT source reposiory.
 	Test string `json:"test"`
+	// Subtest is the name of a subtest that is associated with this test.
+	Subtest string `json:"subtest,omitempty"`
 	// URLs represents a list of bug urls that are associated with
 	// this test.
 	URLs []string `json:"urls,omitempty"`
+	// URLs represents a list of test status that are associated with
+	// this test.
+	Status []TestStatus `json:"status,omitempty"`
```

Comment:
Shall we return the string representation instead? Otherwise, we'd have to at least document the enum values somewhere.

Reply:
FWIW, the current result JSON uses strings (e.g. `status: "FAIL"`)